libcst/_nodes/statement.py

```diff
@@ -906,6 +906,9 @@ def _validate(self) -> None:
                 "A Try statement must have at least one ExceptHandler in order "
                 + "to have an Else"
             )
+        # Check bare excepts are always at the last position
+        if any(handler.type is None for handler in self.handlers[:-1]):
+            raise CSTValidationError("The bare except: handler must be the last one")
```

Comment:
Nit.
```suggestion
            raise CSTValidationError("The bare except: handler must be the last one.")
```

Reply:
@jimmylai I didn't add the trailing period as the errors above don't have it.

The `ImportAlias` statement below does have trailing periods though.

Reply:
Got it. Let's add to both.